front50-sql/src/main/resources/db/changelog/20191030-initial-plugins-schema.yml

```diff
@@ -0,0 +1,47 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-plugins-table
+      author: jonsie
+      changes:
+        - createTable:
+            tableName: plugins
+            columns:
+              - column:
+                  name: id
+                  type: char(255)
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: body
+                  type: longtext
```

Comment:
my sql-neckbeard isn't fabulous but I think it's better to stick a longtext column at the end of the columns list? someone else probably knows better

Reply:
I suppose it can't hurt.  This is a copy paste of other liquibase migration files for a generic object blobs (for example, the exact same as `snapshots`, `notifications`, etc).  But yea, I'll do this.